tests/subsys/bootloader/bl_validation/prj.conf

```diff
@@ -0,0 +1,13 @@
+#
+# Copyright (c) 2018 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+CONFIG_ZTEST=y
+CONFIG_ZTEST_STACKSIZE=12384
```

Comment:
Please check if this is really needed.

Reply:
This will be needed in the future. Will remove here and add when needed.

Reply:
Done